core/services/synchronization/telem/telem_enhanced_ea_mercury.proto

```diff
@@ -5,23 +5,42 @@ option go_package = "github.com/smartcontractkit/chainlink/v2/core/services/sync
 package telem;
 
 message EnhancedEAMercury {
+  uint32 version = 29;
```

Comment:
is `version` here going to correspond to the observation version? so the expected values are going to be either `1`, `2`, or `3`?

Reply:
Yes that is correct